frontend/app/src/components/staking/kraken/KrakenStakingOverview.vue

```diff
@@ -87,7 +87,7 @@ export default defineComponent({
   name: 'KrakenStakingOverview',
   components: { ValueAccuracyHint },
   props: {
-    totalUsd: { required: true, type: Object as PropType<BigNumber> },
+    totalUsd: { required: true, type: BigNumber },
```

Comment:
doesn't this give problems?

Reply:
`type: BigNumber` works well. 
The old implementation throws warning messages.